ncbi_taxonomy/api/utils.py

```diff
@@ -0,0 +1,20 @@
+from django.db import connections
+from ncbi_taxonomy.models import TaxonomyNode
+
+
+class TaxonomyUtils(object):
+
+    @classmethod
+    def fetch_node(cls, taxon_id):
+        node = TaxonomyNode.objects.get(pk=taxon_id)
+        return node
+
+    @classmethod
+    def fetch_descendent_ids(cls, taxon_ids):
+        cursor = connections['ncbi_taxonomy'].cursor()
+        sql = "SELECT n.taxon_id FROM ncbi_taxa_node n  JOIN ncbi_taxa_node parent ON " + \
```

Comment:
Please add a #TODO to use queryset language at some point.

Reply:
Will do - I did consider changing to something like that, but it involves a self join, and it wasn't obvious how to manage that with a queryset command.

Reply:
This is because this is not obvious that I don't ask you to do it right now :-) Thanks!